string-manipulation-2/src/num-vowels.js

```diff
@@ -0,0 +1,16 @@
+/* exported numVowels */
+
+function numVowels(string) {
+  var vowelNumber = 0;
+  var numArray = [];
```

Comment:
You can leave out the array and just increment your `vowelNumber` and return it at the end.